src/openms/source/CHEMISTRY/EnzymesDB.cpp

```diff
@@ -53,12 +53,11 @@ namespace OpenMS
   EnzymesDB::EnzymesDB()
   {
     readEnzymesFromFile_("CHEMISTRY/Enzymes.xml");
```

Comment:
I think I figured out the problem. I'm using a development build of OpenMS, from which I occasionally create installations (snapshots) for production. To do this I set "CMAKE_INSTALL_PREFIX" in the build configuration and run "make install". For my own uses I just create a local build using "make" (but "CMAKE_INSTALL_PREFIX" is still present in the configuration).

The cause of the problem is that the path to "OpenMS/share" is relative to "CMAKE_INSTALL_PREFIX"! Since I haven't updated the installation in a while, that "share" doesn't contain the new "Enzymes.xml" yet. I had a problem before where paths to "OpenMS/share" in the installation still pointed to the "share" in the development build, which led to strange errors. This was fixed, but apparently now I get errors in the other direction.

I think setting the paths to "share" (etc.?) has to be handled differently in the case of "make" (use "share" from the source) and "make install" (use the extra copy of "share" that gets created during installation).